_security-plugin/configuration/yaml.md

```diff
@@ -121,8 +121,57 @@ If you want to run your users' passwords against some validation, specify a regu
 Note that OpenSearch validates only users and passwords created through OpenSearch Dashboards or the REST API.
 
 ```yml
-plugins.restapi.password_validation_regex: '(?=.*[A-Z])(?=.*[^a-zA-Z\d])(?=.*[0-9])(?=.*[a-z]).{8,}'
-plugins.restapi.password_validation_error_message: "Password must be minimum 8 characters long and must contain at least one uppercase letter, one lowercase letter, one digit, and one special character."
+plugins.security.restapi.password_validation_regex: '(?=.*[A-Z])(?=.*[^a-zA-Z\d])(?=.*[0-9])(?=.*[a-z]).{8,}'
+plugins.security.restapi.password_validation_error_message: "Password must be minimum 8 characters long and must contain at least one uppercase letter, one lowercase letter, one digit, and one special character."
+```
+
+## whitelist.yml
+
+You can use `whitelist.yml` to whitelist any endpoints and associated HTTP requests. If enabled, all users except the SuperAdmin are allowed access to only the specified endpoints and HTTP requests, and all other HTTP requests associated with the endpoint are not allowed. For example, if `_cluster/settings` is whitelisted with the GET operation, users are not allowed to submit PUT requests to `_cluster/settings` to update cluster settings.
```

Comment:
I know the file is literally named `whitelist.yml`, but the preferred team here is "allow list." So until the code changes, we want to say "allow list" in prose to refer to entries you add in `whitelist.yml`.